### ISSUE TITLE
Questions about dataset

### ISSUE THREAD

**Issue by pliu19** [iit-cs579/main#185]
Hi, Professor, 

Do we need to crawl the data by ourselves?  Or "edges.txt.gz" should be given? 

Best wishes, 
Ping

**Comment by davidghiurco** [comment]
If you read the a1.py file all the way at the bottom before the main function:
Next, we'll download a real dataset to see how our algorithm performs.
"""
def download_data():
    """
    Download the data. Done for you.
    """
    urllib.request.urlretrieve('http://cs.iit.edu/~culotta/cs579/a1/edges.txt.gz', 'edges.txt.gz')


def read_graph():
    """ Read 'edges.txt.gz' into a networkx **undirected** graph.
    Done for you.
    Returns:
      A networkx undirected graph.
    """
    return nx.read_edgelist('edges.txt.gz', delimiter='\t')

TL;DR: No you do not. The professor did it for you.
